sections/Pools/PoolDurationEnded/PoolDurationEndedSection/PoolDurationEndedSection.tsx

```diff
@@ -36,14 +37,13 @@ interface PoolDurationEndedProps {
 const PoolDurationEnded: FC<PoolDurationEndedProps> = ({ pool, dealID }) => {
 	const { walletAddress, signer } = Connector.useContainer();
 	const { monitorTransaction } = TransactionNotifier.useContainer();
-	const [gasLimitEstimate, setGasLimitEstimate] = useState<GasLimitEstimate>(null);
+	const { gasPrice, setTxState, setTxType } = TransactionData.useContainer();
 
-	const { gasPrice, setGasPrice, txState, setTxState, txType, setTxType } =
-		TransactionData.useContainer();
+	const [inputValue, setInputValue] = useState<number | string>('');
 	const [isMaxValue, setIsMaxValue] = useState<boolean>(false);
-	const [inputValue, setInputValue] = useState(0);
+	const [gasLimitEstimate, setGasLimitEstimate] = useState<GasLimitEstimate>(null);
 
-	useEffect(() => setTxType(TransactionType.Withdraw), []);
+	useEffect(() => setTxType(TransactionType.Withdraw), [setTxType]);
```

Comment:
Same. Always confused about this.

Reply:
we almost for sure do not need the transaction types anymore now that we are splitting it up. in that first component above there are the accept or withdraw types but we already have a boolean that handles that which we can use. There are much simpler ways to manage accept and withdraw and then we can remove this from everywhere

Reply:
Done